static/css/quiz-css.css

```diff
@@ -64,14 +64,14 @@
 
 .option-block code{
     margin-top: -3rem;
-}
+} */
 
-.option-text{
+/* .option-text{
     margin-top: -2rem;
     padding: 0.5rem 0.5rem 0.5rem 0.5rem;
-}
+} */
 
-.option-row:nth-child(1) .option-block:nth-child(1) {
+/* .option-row:nth-child(1) .option-block:nth-child(1) {
     background-color:  #c6f6d5;
     border-color: #2d9068;
 }
```

Comment:
All these classes determine the background (and border) color of the different shaped option block numbers. Finding a way to still have these attributes will fix the "all shapes are black" issue.

Reply:
I'm a dumbass. Sorry, this was a stupid mistake I overlooked. 

Yep it worked.

I have a question before I finalize this commit. I've managed  to clean and make most of those vanilla CSS classes to tailwind classes.  progress-bar.css and quiz-css.css has some classes which are still being used. So the question is, should I just transfer those classes to additional.css or just leave it as it is for now?

Reply:
Awesome! It would be great if you could also transfer these classes to `additional.css` to reduce the amount of styling files. However, this is already a big step forwards so it's also fine if you leave it than we can pick this up in a separate issue.

Reply:
Cool. I'll be submitting the PR tomorrow. Tnx for the help!

Reply:
Thanks for your contribution! Don't forget to update your branch with main before setting it as "Ready for review". As we merge quite some PRs throughout a week your branch is already behind the main branch. Not an issue at all! But it is always nice if its up-to-date before we review it.

You can do this in your IDE by selecting the main branch and choose "merge into ...." or simply perform this action in the GitHub website. The button should be somewhere at the bottom of this PR page.

Reply:
I submitted it as "Ready for review".